assets/components/atoms/link/link.scss

```diff
@@ -41,3 +41,24 @@ a {
 }
 
 // Link collapse, see collapse component
+
+// Link arrow
+.link-arrow {
+  display: flex;
+  justify-content: space-between;
+
+  &:after {
+    content: str-replace(url("data:image/svg+xml;charset=utf8,%3Csvg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 16 16'%3E%3Cpath fill='currentColor' d='M11.586 7L6.293 1.707 7.707.293 15.414 8l-7.707 7.707-1.414-1.414L11.586 9H0V7h11.586z'/%3E%3C/svg%3E"), "#", "%23");
```

Comment:
You mean add the component in the markup? Totally doable but it's a less clean markup then. Otherwise I can just use the SVG file as a background. 

Doing as I did is more practical to play with colors, but since I didn't play with colors, we can totally change it.